delivery_postlogistics_dangerous_goods/models/stock_picking.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+from ..postlogistics.web_service import PostlogisticsWebServiceDangerousGoods
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _generate_postlogistics_label(self, webservice_class=None, package_ids=None):
+        """ Generate post logistic label using specific from this module."""
+        return super()._generate_postlogistics_label(
+            webservice_class=PostlogisticsWebServiceDangerousGoods,
+            package_ids=package_ids,
+        )
```

Comment:
We have to check if `webservice_class` is set before enforce the use of `PostlogisticsWebServiceDangerousGoods` otherwise we are skipping the logic implemented by a user's defined web service passed in the `webservice_class` parameter.

Reply:
@santostelmo is on it

Reply:
fixup pushed